templates/components/itilobject/timeline/form_followup.html.twig

```diff
@@ -124,6 +124,8 @@
                   <div class="col-12 col-xl-5 col-xxl-4 order-first order-md-last pe-0 pe-xxl-auto">
                      <div class="row">
 
+                        {{ include('components/itilobject/timeline/knowledge_item.html.twig', {itilobject: 'followup'}) }}
```

Comment:
IMHO, you should pass a classname here. It would prevent any ambiguity.

```suggestion
                        {{ include('components/itilobject/timeline/knowledge_item.html.twig', {itilobject: 'ITILFollowup'}) }}
```

Reply:
I do not know exactly how it's done for Twig templates, but in PHP code, we try to avoid using hardcoded strings for classnames (`ITILFollowup::class` instead of `'ITILFollowup'`)